web/pgadmin/static/js/components/PgReactDataGrid.jsx

```diff
@@ -158,6 +158,7 @@ export default function PgReactDataGrid({gridRef, className, hasSelectColumn=tru
           renderSortStatus,
           noRowsFallback: <Box textAlign="center" gridColumn="1/-1" p={1}>{noRowsIcon}{noRowsText || gettext('No rows found.')}</Box>,
         }}
+        enableRangeSelection={true}
```

Comment:
Same as above. You should not change the common component default behaviour.

Reply:
yeah agree thanks - have updated so its set via `prop.enableRangeSelection`